ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -281,6 +361,20 @@ export default class TransactionListItemDetails extends PureComponent {
                 primaryCurrency={primaryCurrency}
                 className="transaction-list-item-details__transaction-breakdown"
               />
+              {
+                ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+                transactionNote && transactionNote.length !== 0 && (
```

Comment:
Can we add some snapshot tests for this condition?

Reply:
Added tests